src/tests/fs_tests.rs

```diff
@@ -582,4 +584,29 @@ mod fs_tests {
         assert_eq!(cage.exit_syscall(), 0);
         lindrustfinalize();
     }
+
+    pub fn ut_lind_fs_rmdir() {
+        lindrustinit();
+        let cage = {CAGE_TABLE.read().unwrap().get(&1).unwrap().clone()};
+
+        let path = String::from("/parent_dir/dir");
+        assert_eq!(cage.mkdir_syscall(&String::from("/parent_dir"), S_IRWXA), 0);
```

Comment:
Remove the &String::from call, just use a &str literal

Reply:
Using &str literal directly results in the error message "a directory component in pathname does not exist or is a dangling symbolic link." It seems that we have to use &String::from.

Reply:
Never mind, I just fixed it. Sorry!